exampleoc/opstateoc/update.sh

```diff
@@ -35,7 +35,6 @@ go run ../../generator/generator.go -path=public,deps -output_file=oc.go \
   public/release/models/policy/openconfig-routing-policy.yang \
   public/release/models/lacp/openconfig-lacp.yang \
   public/release/models/system/openconfig-system.yang \
-  public/release/models/lldp/openconfig-lldp.yang \
```

Comment:
Why was this removed please?

Reply:
Because it's already there (see last YANG file). There was a goyang change to error out for duplicate modules.

Reply:
Ah, apologies. I missed this.